arbos/storage/storage.go

```diff
@@ -295,7 +295,8 @@ func (ss *StorageSlot) Get() (common.Hash, error) {
 func (ss *StorageSlot) Set(value common.Hash) error {
 	if ss.burner.ReadOnly() {
 		log.Error("Read-only burner attempted to mutate state", "value", value)
-		return vm.ErrWriteProtection
+		panic("Read-only burner attempted to mutate state")
+		// return vm.ErrWriteProtection
```

Comment:
Did we want to keep this change? I think it's probably good but we should remove the commented out return if so.

Reply:
Fixed